### ISSUE TITLE
Video Bit Rate Reporting - MediaBrowser/MediaInfo Compatibility Mode

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#856]
Originally reported on Google Code with ID 857

```
This feature request relates to previously reported Issue 847.

MediaInfo handles video file bit rate data in a strange way. If the bit rate is four
digits or less (e.g., XXXX), MediaInfo assumes it is in kbps. If the bit rate in the
XML is presented as five or more digits, MediaInfo assumes it is in bps. 

See

http://community.mediabrowser.tv/permalinks/1312/how-do-i-configure-and-use-the-mediainfo-plugin

This explains why DVD's report correctly with MCM because the DVD bit rate is typically
5,000 kbps, so the MCM XML says "5,000," which is four digits or less, and thus MediaInfo
understands it is in kbps and displays it as such in MB. Everything is consistent.

On the other hand, Blu-Ray's typically have a bit rate of 25,000 kbps. MCM writes the
XML as "25,000," but because this is five digits long MediaInfo assumes it is now being
presented in bps. Thus MB/MI displays "25 kbps." 

In short, MCM is consistently writing the XML bit rate in kbps. But since MI switches
from kbps to bps with five or more digits, MCM needs to report the bit rate of Blu-rays
in bps (e.g., 25,000,000).

Conclusion - feature should be added for MB/MI compatibility that causes MCM to save
Blu-Ray bit rates to the XML in bps rather than kbps.

```

Reported by `pliesj@yahoo.com` on 2011-11-17 16:54:25
